docs/docs.md

```diff
@@ -80,10 +72,10 @@ Make a GET request to the Dataverse API with currently added query options.
 **Error Simplification Available**:
 This and the other HTTP-methods are decorated with a decorator,
 that can take some extra options: `simplify_errors` (If set `True`, will simplify all errors occurring
-during the execution of the function to just a single DynamicsException with a default
+during the excecution of the function to just a single DynamicsException with a default
```

Comment:
Bunch of the misspellings corrected in the previous PR are back.

Reply:
Problem caused by me needing to sync my own main branch to catch up with yours - resolved. Thanks for pointing it out.